src/borkdude/deps.clj

```diff
@@ -216,6 +216,47 @@ function Get-StringHash($str) {
     (unzip dest (.getPath deps-clj-config-dir))
     (.delete dest)))
 
+(def ^:private authenticated-proxy-re #".+:.+@(.+):(\d+).*")
```

Comment:
Why dynamic vars?

Reply:
Not dynamic, just private. I probably had them used in more than 1 place originally and hence had them in a constant def. The final version only has them in 1 place so can remove the def's and put them where they're used. Got spot.

Reply:
Sorry, misread. This is fine.